Sources/FeaturevisorSDK/Conditions.swift

```diff
@@ -95,6 +95,17 @@ public func conditionIsMatched(condition: PlainCondition, context: Context) -> B
                     return false
             }
 
+        // string, [strings]
+        case (.string(let valueInAttribute), .array(let valuesInCondition)):
+            switch op {
+                case .in:
```

Comment:
not directly related to this PR, but I couldn't find regular string/string operators in this function.

Reply:
I noticed you have tests for string equals/notEquals, so all good. I just couldn't find the source code.

Reply:
No worries, it's there:
```
        // string, string
        case (.string(let valueInAttributes), .string(let valueInCondition)):
```